src/components/DeploymentParameters.vue

```diff
@@ -30,11 +31,13 @@
 
   const parameters = computed({
     get() {
-      const source = { values: props.deployment.parameters, schema: parameterOpenApiSchema.value }
+      const source = { values: props.modelValue, schema: parameterOpenApiSchema.value }
```

Comment:
Should this be taking both deployment and model value into account?

Reply:
No. We're extracting parameters from the deployment at the form level so they're passed as part of the model value. 

The only place where there's a difference between the deployment parameters and the model value is in the custom run form where we need to consider the parameter value passed as props because of the "copy to new run" option. We do the handling of those in the form and then pass the updated parameters to this component.

Reply:
Got it - thanks for the explanation!